src/libraries/CurrencyDelta.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.20;
+
+import {Currency} from "../types/Currency.sol";
+
+library CurrencyDelta {
+    uint256 constant CURRENCY_DELTA_SLOT = uint256(keccak256("CurrencyDelta")) - 1;
```

Comment:
Instead of this computing a hash every time we can just use the hex val and comment above?

Reply:
oh yeeaaah nice haha, i made that PR after this